apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -3352,14 +3366,43 @@ public enum CIGroupInvitationStatus: String, Decodable {
 
 public enum RcvDirectEvent: Decodable {
     case contactDeleted
+    case profileUpdated(fromProfile: Profile, toProfile: Profile)
 
     var text: String {
         switch self {
         case .contactDeleted: return NSLocalizedString("deleted contact", comment: "rcv direct event chat item")
+        case let .profileUpdated(fromProfile, toProfile): return profileUpdatedText(fromProfile, toProfile)
+        }
+    }
+
+    private func profileUpdatedText(_ from: Profile, _ to: Profile) -> String {
```

Comment:
```suggestion
    private func profileUpdatedText(_ from: Profile, _ to: Profile) -> LocalizeStringKey {
```

or am I missing something?